ios/Classes/Common/FlutterCoder.swift

```diff
@@ -524,7 +524,7 @@ extension Float: FlutterConvertible {
 extension Int: FlutterConvertible {
     static func fromFlutterValue(_ value: Any) -> Int {
         guard let num = value as? NSNumber else {
-            fatalError("TODO: Throw error here")
+            fatalError(.castingToNumberFailed)
```

Comment:
Could this method be made to throw? And the same needs to be asked for similar functions.